networkx/algorithms/bipartite/tests/test_matrix.py

```diff
@@ -39,6 +39,11 @@ def test_biadjacency_matrix_order(self):
         M = bipartite.biadjacency_matrix(G, X, Y, weight="weight")
         assert M[1, 2] == 2
 
+    def test_biadjacency_matrix_empty_graph(self):
+        G = nx.empty_graph(2)
+        M = nx.bipartite.biadjacency_matrix(G, [0])
+        assert M == sp.sparse.coo_matrix(([], ([], [])), shape=(1, 1))
```

Comment:
It's not particularly clear what sparse matrix will actually result from this call; I'd vote for comparing the result to the expected array directly instead. Also, NX now uses sparse *arrays* not matrices:

```suggestion
        assert np.array_equal(M.toarray(), np.array([[0]]))
```

Reply:
I agree, this does make it more clear. Thanks for helping me!!